platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -131,7 +131,7 @@ export default class DisplaySetService extends PubSubService {
       : [...this.getDisplaySetCache().values()];
 
     const displaySet = displaySets.find(ds => {
-      return ds.images && ds.images.some(i => i.SOPInstanceUID === sopInstanceUID);
+      return ds.instances?.some(i => i.SOPInstanceUID === sopInstanceUID);
```

Comment:
Are we just utilizing the instances instead of images, or are we deleting images too? If so, we need to add this information to the breaking change page.

Reply:
Just utilizing instances instead of images because video display sets don't have the images, whereas instances is the canonical name for any instance.